packages/pwa-kit-react-sdk/src/ssr/server/express.js

```diff
@@ -1682,11 +1685,13 @@ const applyPatches = once((options) => {
     // Patch the http.request/get and https.request/get
     // functions to allow us to intercept them (since
     // there are multiple ways to make requests in Node).
-    const getAppHost = () => options.appHostname
-    http.request = outgoingRequestHook(http.request, getAppHost)
-    http.get = outgoingRequestHook(http.get, getAppHost)
-    https.request = outgoingRequestHook(https.request, getAppHost)
-    https.get = outgoingRequestHook(https.get, getAppHost)
+
+    // TODO: Figure out what the significance of making this a function.
+    // const getAppHost = () => options.appHostname
```

Comment:
@bendvc once upon a time this code may have been used, but it appears to be dead code now.